packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -216,7 +227,7 @@ export default class MediaEmbedEditing extends Plugin {
 			// Upcast semantic media.
 			.elementToElement( {
 				view: {
-					name: 'oembed',
+					name: new RegExp( `^(${ elementNames.join( '|' ) })$` ),
```

Comment:
I think that plain function will be much more readable instead or a RegExp and also `elementNames` array could be skipped then:

```javascript
view: element => [ 'oembed', preferredElementName ].includes( element.name ) && element.getAttribute( 'url' ) ?
	{ name: true } :
	null,
```

Reply:
I used the snippet here in d6f2c91